backend/main/models/proposal.go

```diff
@@ -35,6 +35,7 @@ type Proposal struct {
 	Timestamp            string                  `json:"timestamp" validate:"required"`
 	Composite_signatures *[]s.CompositeSignature `json:"compositeSignatures" validate:"required"`
 	Computed_status      *string                 `json:"computedStatus,omitempty"`
+	Snapshot             s.LatestBlockHeight     `json:"snapshot,omitempty"`
```

Comment:
i think this should be a pointer, if there is a possibility of the field being empty